### PR TITLE
Add Unit Tests to hit 70% coverage

### PR DESCRIPTION
**Pull request by MalanchukA**
- Add some minor styling to Calendar Component
- Replace all .css with styled components

### CODE REVIEW

File: src/Components/Guests.jsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ class Guests extends React.Component{
             <Form onClick={this.props.changeVisibility}>
         <GuestCount> {adults + children} {adults + children === 1 
         ? "guest" : "guests"}{infants === 0 ? '' : `, ${infants} ${infants === 1 
-            ? "infant" : "infants"}`}</GuestCount><DropDown>\/</DropDown>
+            ? "infant" : "infants"}`}</GuestCount><DropDown><svg width="24" height="24" xmlns="http://www.w3.org/2000/svg" fill-rule="evenodd" clip-rule="evenodd"><path d="M23.245 4l-11.245 14.374-11.219-14.374-.781.619 12 15.381 12-15.391-.755-.609z"/></svg></DropDown>
```
Comment (f3lixding) [inline src/Components/Guests.jsx]: I see that you surrendered and used svg in the end.

Comment (MalanchukA) [reply src/Components/Guests.jsx]: Thanks for giving a tip. I tried and it indeed looks better

File: __tests__/App.test.jsx [hunk 2]
```diff
@@ -29,12 +29,26 @@ describe('<App/>', () => {
...
+
+  it('toggles the state is Clicked for Guests', () => {
+    const wrapper = create(<App />);
+    const instance = wrapper.getInstance();
+    expect(instance.state.guestsClicked).toEqual(false);
+    instance.changeGuestsVisible();
+    expect(instance.state.guestsClicked).toEqual(true);
+    instance.changeGuestsVisible();
+    expect(instance.state.guestsClicked).toEqual(false);
+  });
```
Comment (f3lixding) [inline __tests__/App.test.jsx]: Maybe it's the way github is presenting the changes, but it really looks like there are some broken parenthesis here...  Or do you not need the 'it' to be within the describe for it to run?

Comment (MalanchukA) [reply __tests__/App.test.jsx]: I think this is how github presents it. You should have 'it' inside describes, otherwise you will get error.

**Pull request merged**